src/platform/f4xx/stm32f446xx.cpp

```diff
@@ -42,7 +42,7 @@ void stm32f4xx_init() {
     */
     RCC_ClkInitStruct.ClockType = RCC_CLOCKTYPE_HCLK | RCC_CLOCKTYPE_SYSCLK
         | RCC_CLOCKTYPE_PCLK1 | RCC_CLOCKTYPE_PCLK2;
-    RCC_ClkInitStruct.SYSCLKSource = RCC_SYSCLKSOURCE_PLLCLK;// SYSCLK at 8 MHz (HSI CLK)
+    RCC_ClkInitStruct.SYSCLKSource = RCC_SYSCLKSOURCE_PLLCLK;// SYSCLK at 50 MHz (HSI CLK)
```

Comment:
it currently is not used anywhere in f4, and in f3 it is only used in the timer cpp.  Reason it isn't used in f4 timer is cause Heller said to just use SystemCoreClock instead of this variable.

Reply:
Just because the variable isn't used doesn't mean it won't break things. This is affecting the hardware configuration, so it may break things that previously worked in hardware. Any thoughts @mjh9585?

Reply:
Just let me know what yall think. I'll revert or w/e Matt and Matt suggest 🐸

Reply:
I am confused what we are referring to since the code referenced is just changing a comment.  If you are referring to CLK_SPEED in stm32fxx.hpp then that is fine as it was already wrong since the chip is running at 50MHz, not 8MHz. Additionally for the timer calculations, Travis is using SystemCoreClock which is calculated when the system clock gets update and is a much better source then a hard coded value.

Reply:
In my head, I was assuming that he actually changed something other than the comment to change the frequency, but if this is just updating documentation to be more accurate, I'm fine with it